src/interfaces/INonfungiblePositionManager.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.24;
+
+import {PoolKey} from "@uniswap/v4-core/src/types/PoolKey.sol";
+
+enum Actions {
+    MINT,
+    BURN,
+    INCREASE,
+    DECREASE,
+    // Any positive delta on a currency will be sent to specified address
+    CLOSE_CURRENCY
+}
+
+interface INonfungiblePositionManager {
```

Comment:
Can we make this IV4NonFungiblePositionManager or something? So it's not identical to the v3 one